tox.ini

```diff
@@ -62,7 +52,7 @@ commands =
 deps =
   -r requirements/type-check.txt
 commands_pre =
-  uv pip install --reinstall .
+  uv pip install --reinstall .[pg]
```

Comment:
we can probably move the pg stuff into type-check.txt so it becomes part of the CI cache key